tutorials/compute-managed-instance-groups-dashboard/index.md

```diff
@@ -0,0 +1,182 @@
+Simple MIG Dashboard
+====================
+
+Introduction
+------------
+
+Simple MIG dashboard is a tool for monitoring the state of instances of
+a Managed Instance Group. For each instance it visualizes its current
+action (e.g. *CREATING*, *RESTARTING*) and its Instance Template. It can be
+especially useful to monitor progress of a rolling update: it shows how
+the state (including health) of each instance is changing over time.
+
+This tutorial explains how to set up and use Simple MIG dashboard on
+your local machine with your Google Cloud Platform project. It walks you
+through the process of setting up a GCP web application and granting it
+all necessary permissions to access your project data.
+
+Simple MIG dashboard is implemented in JavaScript using Angular
+framework. It uses Google Charts library for visualization purposes and
+Google API Client Library for JavaScript to communicate with Google
+Cloud Platform.
+
+Before you begin
+----------------
+
+In this tutorial, we assume that you have a Google Cloud Platform
+project with an existing Managed Instance Group, which you want to
```

Comment:
change "an existing Managed Instance Group, which you want to monitor."
to
"an existing Managed Instance Group that you want to monitor."